lang/cs/Org.Apache.REEF.IO/CacheLevel.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace Org.Apache.REEF.IO
+{
+    /// <summary>
+    /// The level at which data is cached.
+    /// </summary>
+    public enum CacheLevel
```

Comment:
Can you please add some `0`s to the constants in here? That way, we can slide `SSD` in between memory and disk at some point.